.github/workflows/ruby.yml

```diff
@@ -47,6 +47,14 @@ jobs:
           ruby: '3.0'
         - rails: '8.0.0'
           ruby: '3.1'
+        - rails: '8.1.0'
```

Comment:
Please update comment above by adding this line:
`# Excludes Rails 8.1 on Ruby 2.6, 2.7, 3.0, 3.1`